ci-operator/jobs/redhat-appstudio/infra-deployments/redhat-appstudio-infra-deployments-main-presubmits.yaml

```diff
@@ -15,7 +15,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-deployment
     rerun_command: /test appstudio-e2e-deployment
-    skip_if_only_changed: \.md$
+    skip_if_only_changed:  ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
Seems like the extra space is making the PR check jobs to fail. Otherwise you have my approval ;-)